brian2/codegen/translation.py

```diff
@@ -349,8 +345,6 @@ def make_statements(code, variables, dtype, optimise=True, blockname=''):
     for line in lines:
         stmt = line.statement
         read = line.read
-        write = line.write
-        will_read = line.will_read
         will_write = line.will_write
```

Comment:
And same here.

Reply:
It's true that there was more consistency here before... I wonder whether it wouldn't be clearer after all to only keep `stmt = line.statement` (because this is used several times and would get unwieldy otherwise), and use `line.read` and `line.will_write` directly, since they are only used once or twice.

Reply:
Maybe, although I don't have very strong feelings about it - so whatever you prefer!

Reply:
I went ahead with the change. Not the most important change ever, though.